pinot-transport/src/test/java/org/apache/pinot/transport/perf/ScatterGatherPerfServer.java

```diff
@@ -152,17 +150,15 @@ public MyRequestHandler(String response, CountDownLatch responseHandlingLatch, l
     }
 
     @Override
-    public ListenableFuture<byte[]> processRequest(ChannelHandlerContext channelHandlerContext, ByteBuf request) {
-      byte[] b = new byte[request.readableBytes()];
-      request.readBytes(b);
+    public ListenableFuture<byte[]> processRequest(byte[] request) {
       if (null != _responseHandlingLatch) {
         try {
           _responseHandlingLatch.await();
         } catch (InterruptedException e) {
           e.printStackTrace();
```

Comment:
Can we put this as `LOGGER.error`?

Reply:
Good point

Reply:
This class is just for perf benchmark purpose (not production code)